Processing/test_yOCTFindTissueSurface.m

```diff
@@ -14,8 +14,10 @@ function createDummyDataset(testCase)
             % Create a dummy dataset with a scattering interface
             testCase.simulatedSurfacePositionZ_pix = 500;
             speckleField = zeros(1024,100,200)+10; %z,x,y
-            speckleField(testCase.simulatedSurfacePositionZ_pix,:,:) = 1000;
-    
+
+            speckleField(testCase.simulatedSurfacePositionZ_pix:end, :, :) = ...
+                10 + 990 * abs(randn(1024 - testCase.simulatedSurfacePositionZ_pix + 1, 100, 200));
```

Comment:
Please avoid using "randn" - random numbers are not good for testing because if something fails due to random value it's hard to reproduce.

Reply:
If you think random numbers reproduce the results better use rng(1); before generating the random numbers.
This will make sure that random numbers start from the same seed (1 in this case)